lib/initializeLocalDb.ts

```diff
@@ -1,4 +1,5 @@
 import { LevelModel, RecordModel, ReviewModel, UserModel, WorldModel } from '../models/mongoose';
+
```

Comment:
looks like your auto-formatter adds a space between multi-imports and single imports